deployments/deployment.yaml

```diff
@@ -932,6 +951,8 @@ parameters:
     value: "0 8 * * 4"
   - name: NIGHTLY_CRON_JOB
     value: "0 0/1 * * *"
+  - name: DAILY_CRON_JOB
+    value: "0 1 * * *"
```

Comment:
would it make sense to call this something like  CLEANUP_CRON_EXP  ? .  I say this as the previous 'nightly' cron is no longer nightly in the different environments

Reply:
yeah, that's a good point